src/checkers/inference/CheckerFrameworkUtil.java

```diff
@@ -7,9 +7,9 @@
 
 public class CheckerFrameworkUtil {
 
-    public static boolean invokeCheckerFramework(String[] args, PrintWriter outputCapture) {
+    public static Result invokeCheckerFramework(String[] args, PrintWriter outputCapture) {
         Main compiler = new Main("javac", outputCapture);
         Result compilerResult = compiler.compile(args);
-        return compilerResult == Result.OK;
+        return compilerResult;
```

Comment:
Instead of returning either success or failure, return the raw compiler result and let result handler to decide what action to take.